packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -63,22 +62,7 @@ struct AnonymousQueryInfo {
     // strings well?
   6: optional string code,
   // TODO should this be optional or always present? or not here at all?
-  7: optional Location location,
-}
-
-struct Location {
-  1: required string url,
-  2: required Range range,
-}
-
-struct Range {
-  1: required Position start,
-  2: required Position end,
-}
-
-struct Position {
-  1: required i32 line,
-  2: required i32 character,
+  7: optional DocumentLocation location,
 }
 
 struct Schema {
```

Comment:
Should `DimensionInfo` and `MeasureInfo` be merged into `AtomicFieldInfo` with a `field_type` field?

Reply:
MToy votes that DimensionInfo and MeasureInfo is good, due to convenience and that the number of different types is small.